TEP002_model_hdf5.rst

```diff
@@ -0,0 +1,62 @@
+TEP002: Meaningful title
+========================
+
+Status
+======
+
+ **Discussion**
+
+Responsible
+===========
+
+TBD
+
+Branches and Pull requests
+==========================
+
+N/A
+
+Description
+===========
+
+Currently, Tardis has the capability to store a number of important properties
+of the model in an HDF5 file. However, the hdf5 file is currently not
+self-contained, i.e. some stored properties require information which is
+currently not stored. For example:
+
+* last_line_interaction_in_id stores the ids of the last line transition in
+  which the packet was lastly absorbed. The details of this transition, i.e.
+  wavelength, atom number, ionisation state, etc. are stored in the
+  atomic_data.lines pandas DataFrame which is not stored
+
+There are also many important model properties, which are particularly useful
+for diagnostic post processing of the run but which are not stored.
+
+* unfiltered packet information together with LF wavelength before and after
+  the last interaction (needed to construct Kromer plots)
+* transitions probabilities (as seen in Issue #455, these are quite important)
+
+As a consequence, there should be the capability of storing the full model
+information in an HDF5 file. This capability may be an additional feature,
+living next to the possibility to store a minimal set of "essential"
+information about the run (similar to what the to_hdf5 currently does). As an
+alternative the to_hdf5 method may be rewritten to operate in two modes,
+minimal and detailed.
+
+Implementation
+==============
+
+N/A
```

Comment:
Suggestion for text:
`For the plasma there is a relatively straightforward way of accomplishing this: Give each property a method to write its current value to an HDF5 file. Some issues need to be looked at like storing single float values, but there are some proposed solutions already for this in the current PR`